app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md

```diff
@@ -0,0 +1,321 @@
+* you have been resident in the UK, EEA or Switzerland throughout the 3 years before the start of your course
+
+you will generally be eligible for a tuition fee loan.
+
+### Trainee teachers from outside the European Economic Area (EEA) or Switzerland
+
+To be eligible for financial support, you must usually have settled status (for example, indefinite leave to remain) and have been living in the UK, Guernsey, Jersey or the Isle of Man throughout the 3 years immediately before the start of your course. Exceptions apply in some cases – for example if you have been recognised as a refugee.
+
+### Refugees and asylum seekers
+
+If you have been recognised as a refugee by the UK government, you, your spouse or civil partner and your children, are eligible for financial support for teacher training.
+
+You will also need:
+
+* to have been living in the UK since you were recognised as a refugee
+* to be living in England when your teacher training course starts
+
+If you are an asylum seeker you will not be eligible for financial support, but if you are recognised as a refugee after your course has started, you may be able to claim financial support to complete your studies.
+
+Financial support can include:
+
+* help with living expenses (a maintenance loan)
+* help with course fees (a tuition fee loan)
+
+If, as a refugee, you are eligible for help with course fees, you may also be eligible for a bursary or scholarship.
+
+## Types of financial support available
+
+If you have the right immigration status, you could be eligible for the following types of financial help from the UK government.
+
+### Tuition fee loan
+Your tuition fees are paid in full, with the money going directly to your teacher training provider. This is a loan you have to pay back, but you’ll only start your repayments when your income is over a certain amount – for the 2023 to 2024 financial year, this is £24,996 per year.
+
+### Maintenance loan
+Money is paid directly into your bank account to help with your living costs. This is a loan you have to pay back, but you’ll only start your repayments when your income is over a certain amount – for the 2023 to 2024 financial year, this is £24,996 per year.
+
+### Home fee rate
+If you meet the requirements for a tuition fee loan, or tuition fee loan and maintenance loan, you will be charged fees at the rate set for UK candidates (set at a maximum of £9,250 for a full-time course).
+
+### Bursaries or scholarships 
+Physics and languages bursaries and scholarships are open to all qualifying non-UK candidates, regardless of their immigration status.
+However, other bursaries and scholarships are only available to candidates who meet the eligibility requirements for a tuition fee loan, or tuition fee and maintenance loan.
+If you are awarded a bursary or scholarship, the money is paid directly into your bank account and you can use it as you wish. You do not have to pay this money back.
```

Comment:
Do we need to put this info onto a new line just to break up this big block of text?